Scoop/Controller/Home/NotificationsViewController.swift

```diff
@@ -106,7 +106,14 @@ extension NotificationsViewController: UITableViewDataSource {
     
     func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         // TODO: Make this dynamic. Works ok for now, but .automaticDimension didn't really work.
-        allRequests[indexPath.row].approved ? 100 : 110
+        if let approved =  allRequests[indexPath.row].approved {
+            if approved {
+                return 100
+            } else {
+                return 110
+            }
+        }
+        return 100
```

Comment:
psa ^ these are my changes and they're kinda scuffed rn but i will fix it in my next PR